docs-site/src/pages/docs/getting-started.js

```diff
@@ -42,7 +42,7 @@ export default () => {
               Pages are picked up from all files under the <code>src/pages</code> directory.
             </p>
             <p>
-              Add a new source file for the app.
+              Add a new source file for the containing a route component for the path <code>/</code>.
```

Comment:
This doesn't read quite clearly to me...?

Reply:
👍